core/templatetags/image_tags.py

```diff
@@ -8,3 +9,14 @@
 def progressive_image(webp, src, alt_text=''):
     context = {'src': src, 'webp': webp, 'alt_text': alt_text}
     return render_to_string('core/widgets/progressive_image.html', context)
+
+
+class FullStaticNode(static.StaticNode):
+    def url(self, context):
+        request = context['request']
+        return request.build_absolute_uri(super().url(context))
+
+
+@register.tag
+def get_image_absolute_uri(parser, token):
```

Comment:
why not just use `%static%`?

Reply:
opengraph needs the full path to the image. currently we're using `<meta property="og:image" content="{% static 'images/icons/opengraph-image.png' %}">` which produces `<meta property="og:image" content="/static/images/icons/opengraph-image.f86f1d0dd106.png">`

Reply:
ah yes this is an annoying thing to solve with django:
- if we serve images from the webserver (local dev) then `{% static ... %}` gives a relative url.
- if we use AWS as the storage backend (prod, staging, heroku dev) then `{% static ... %}` gives an absolute url pointing to the s3 bucket (as defined by `AWS_S3_CUSTOM_DOMAIN` in `settings.py`.

Unfortunately `request.build_absolute_uri` will create a absolute url for the current domain, not the s3 domain that images are served from if s3 storage backend is used.